_woc/WoCabulary.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "WoCabulary - My Woc Blog"
+excerpt: "This is the inital blog post to derive a generalized interest for the project of Text Sentiment Analysis."
+toc: true
+toc_sticky: true
+toc_label: "Contents"
+
+categories:
+  - woc
+
+tags:
```

Comment:
Write your own tags!
It can be your college name or your fields of interests. Like NLP, Python or others.